postgres/messages/row_description.go

```diff
@@ -277,6 +277,8 @@ func VitessTypeToObjectID(typ query.Type) (int32, error) {
 		return OidDate, nil
 	case query.Type_NULL_TYPE:
 		return OidText, nil // NULL is treated as TEXT on the wire
+	case query.Type_ENUM:
+		return OidText, nil // TODO: temporary solution until we support CREATE TYPE
```

Comment:
It doesn't look like these are actually being referenced at all? Are we able to remove them?

Reply:
Ah I just unskipped the test that uses these.

Reply:
Ohh I see, didn’t catch that.